### PR TITLE
SBS-2920 Line range for seismics proto

### PR DESCRIPTION
**Pull request by BenChand**

### CODE REVIEW

File: cognite/seismic/protos/v1/seismic_service_datatypes.proto [hunk 1]
```diff
@@ -26,8 +26,8 @@ message LineBasedVolume {
 }
 
 message LineRange {
-    google.protobuf.Int32Value from_line = 1;
-    google.protobuf.Int32Value to_line = 2;
+    LineDescriptor inline = 1;
```
Comment (amorken) [inline cognite/seismic/protos/v1/seismic_service_datatypes.proto:29]: @skairunner These used in the SDK yet?

Comment (amorken) [reply cognite/seismic/protos/v1/seismic_service_datatypes.proto:29]: (That is, the previous version of these fields)

Comment (skairunner) [reply cognite/seismic/protos/v1/seismic_service_datatypes.proto:29]: They have never been filled in (so never have been not null), so I think it's safe to overwrite.

Comment (amorken) [reply cognite/seismic/protos/v1/seismic_service_datatypes.proto:29]: :+1:

**Review by amorken — APPROVED** [review]

**Pull request merged**